go/billing/api.py

```diff
@@ -228,34 +244,49 @@ def create_transaction_interaction(self, cursor, account_number,
         session_credits = MessageCost.calculate_session_credit_cost(
             session_cost, markup_percent)
 
+        session_unit_credits = MessageCost.calculate_session_unit_credit_cost(
+            session_unit_cost, markup_percent)
```

Comment:
It seems a bit weird to be storing ``session_unit_credits`` because there isn't really a transaction cost assosiate with this, right? There's just ``session_credits`` and ``session_len_credits``?

This may affect #1171 too.